lib/simplifyExpression/basicsSearch/index.js

```diff
@@ -3,60 +3,18 @@
  * These are always the first simplifications that are attempted.
  */
 
+const basic_rules = require('./basicRules.js');
 const Node = require('../../node');
 const TreeSearch = require('../../TreeSearch');
 
-// const rearrangeCoefficient = require('./rearrangeCoefficient');
-// const reduceExponentByZero = require('./reduceExponentByZero');
-const reduceMultiplicationByZero = require('./reduceMultiplicationByZero');
-// const reduceZeroDividedByAnything = require('./reduceZeroDividedByAnything');
-const removeAdditionOfZero = require('./removeAdditionOfZero');
-// const removeDivisionByOne = require('./removeDivisionByOne');
-// const removeExponentBaseOne = require('./removeExponentBaseOne');
-// const removeExponentByOne = require('./removeExponentByOne');
-// const removeMultiplicationByNegativeOne = require('./removeMultiplicationByNegativeOne');
-// const removeMultiplicationByOne = require('./removeMultiplicationByOne');
-// const simplifyDoubleUnaryMinus = require('./simplifyDoubleUnaryMinus');
-
-// const SIMPLIFICATION_FUNCTIONS = [
-//   // multiplication by 0 yields 0
-//   reduceMultiplicationByZero,
-//   // division of 0 by something yields 0
-//   reduceZeroDividedByAnything,
-//   // ____^0 --> 1
-//   reduceExponentByZero,
-//   // Check for x^1 which should be reduced to x
-//   removeExponentByOne,
-//   // Check for 1^x which should be reduced to 1
-//   // if x can be simplified to a constant
-//   removeExponentBaseOne,
-//   // - - becomes +
-//   simplifyDoubleUnaryMinus,
-//   // If this is a + node and one of the operands is 0, get rid of the 0
-//   removeAdditionOfZero,
-//   // If this is a * node and one of the operands is 1, get rid of the 1
-//   removeMultiplicationByOne,
-//   // In some cases, remove multiplying by -1
-//   removeMultiplicationByNegativeOne,
-//   // If this is a / node and the denominator is 1 or -1, get rid of it
-//   removeDivisionByOne,
-//   // e.g. x*5 -> 5x
-//   rearrangeCoefficient,
-// ];
-
-const SIMPLIFICATION_FUNCTIONS = [
-  // multiplication by 0 yields 0
-  reduceMultiplicationByZero,
-  // If this is a + node and one of the operands is 0, get rid of the 0
-  removeAdditionOfZero,
-];
+const SIMPLIFICATION_FUNCTIONS = Object.keys(basic_rules);
 
 const search = TreeSearch.preOrder(basics);
 
 // Look for basic step(s) to perform on a node. Returns a Node.Status object.
 function basics(node) {
   for (let i = 0; i < SIMPLIFICATION_FUNCTIONS.length; i++) {
```

Comment:
you can just iterate through the keys here, I don't think you need to define `SIMPLIFICATION_FUNCTIONS` anymore